PENDING_RELEASE_NOTES.md

```diff
@@ -1,8 +1,10 @@
 
-## Version 20.11.0
+## Version 20.12.0
```

Comment:
When you get to the point of resolving the merge conflict in this file, I don't think the version number should have changed. The last release was 20.10.0.